### DISCUSSION TITLE
contentlayer issue

### DISCUSSION THREAD

**Discussion by Adugnatad** [dubinc/dub#264] [Q&A]
Is anyone facing this error when trying to run the dev script

import { ConsoleSpanExporter, SimpleSpanProcessor } from "@opentelemetry/sdk-trace-base";
         ^^^^^^^^^^^^^^^^^^^
SyntaxError: Named export 'ConsoleSpanExporter' not found. The requested module '@opentelemetry/sdk-trace-base' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from '@opentelemetry/sdk-trace-base';
const { ConsoleSpanExporter, SimpleSpanProcessor } = pkg;

    at ModuleJob._instantiate (node:internal/modules/esm/module_job:123:21)
    at async ModuleJob.run (node:internal/modules/esm/module_job:189:5)

**Comment by elie222** [comment]
This fixes it for now:
https://github.com/contentlayerdev/contentlayer/issues/506#issuecomment-1625210487
